datasets/world-happiness/metadata.json

```diff
@@ -0,0 +1,673 @@
+{
+  "@context": {
+    "@language": "en",
+    "@vocab": "https://schema.org/",
+    "column": "ml:column",
+    "extract": "ml:extract",
+    "field": "ml:field",
+    "fileProperty": "ml:fileProperty",
+    "format": "ml:format",
+    "includes": "ml:includes",
+    "isEnumeration": "ml:isEnumeration",
+    "jsonPath": "ml:jsonPath",
+    "ml": "http://mlcommons.org/schema/",
+    "parentField": "ml:parentField",
+    "path": "ml:path",
+    "recordSet": "ml:recordSet",
+    "references": "ml:references",
+    "regex": "ml:regex",
+    "repeated": "ml:repeated",
+    "replace": "ml:replace",
+    "sc": "https://schema.org/",
+    "separator": "ml:separator",
+    "source": "ml:source",
+    "subField": "ml:subField",
+    "transform": "ml:transform",
+    "data": { "@id": "ml:data", "@type": "@json" },
+    "dataType": { "@id": "ml:dataType", "@type": "@vocab" }
+  },
+  "citation": "None",
+  "license": "https://creativecommons.org/publicdomain/zero/1.0/",
+  "url": "https://www.kaggle.com/datasets/unsdsn/world-happiness",
+  "distribution": [
+    {
+      "contentUrl": "https://www.kaggle.com/api/v1/datasets/download/unsdsn/world-happiness",
```

Comment:
Interesting. I thought a test would fail if the keys don't follow a pre-defined order. (This is to simplify the git changes when we migrate JSON-LD in bulk.)